src/shared/components/palette/dynamic-list/read-dynamic-list-field.component.ts

```diff
@@ -0,0 +1,11 @@
+import { AbstractFieldReadComponent } from '../base-field/abstract-field-read.component';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ccd-read-dynamic-list-field',
+  template: '<span *ngIf="caseField.value" class="text-16">' +
+    '{{caseField.value.default.code | ccdDynamicList:caseField.value.dynamic_list_items}}' +
+    '</span>',
+})
+export class ReadDynamicListFieldComponent extends AbstractFieldReadComponent {
```

Comment:
do we really need to introduce new palette components for Dynamic List?
Can't we reuse FixedList components? 
In both cases we just have to display a dropdown so what we render on the page is exactly the same

Reply:
removed